plugins/CoreAdminHome/Commands/PurgeLegacyAnnotations.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\CoreAdminHome\Commands;
+
+use Piwik\Common;
+use Piwik\Container\StaticContainer;
+use Piwik\Plugin\ConsoleCommand;
+use Piwik\Updater;
+use Piwik\Updater\Migration\Factory as MigrationFactory;
+
+/**
+ * Command to purge annotations from option table after migrating them to a separate db table.
+ */
+class PurgeLegacyAnnotations extends ConsoleCommand
+{
+    protected function configure()
+    {
+        $this->setName('core:matomo550-purge-legacy-annotations');
+        $this->setDescription('Purge legacy annotations from option table after migrating them to a separate annotations table. ');
+        $this->setDescription('Only needed for Matomo 5.5.0-b1 upgrade');
```

Comment:
Was that intended to be a concatenated description like `core:matomo550-migrate-annotations`?

Right now it only shows the second `setDescription` value.

Reply:
good catch